samples/drivers/i2s/output/boards/nucleo_g431kb.conf

```diff
@@ -0,0 +1 @@
+CONFIG_HEAP_MEM_POOL_SIZE=4192
```

Comment:
BTW, where do we use it?

From what I see we should do allocations from message queue (statically defined in sai driver) and from slab (statically defined in samples/drivers/i2s/output/src/main.c).

Where do we use system heap?